recipes/virtualship/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "virtualship" %}
+{% set version = "0.0.3.dev44" %}
```

Comment:
Can you set this to a release, please?

Reply:
Or follow CFEP-05. See https://github.com/conda-forge/cfep/blob/main/cfep-05.md